### DISCUSSION TITLE
PaddleOCR/applications/overview

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13762] [Q&A]
# PaddleOCR/applications/overview

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/applications/overview.html

**Comment by wangjiannan98** [comment]
请问现在这些模型是下载不了了吗？

**Reply by GreatV** [reply]
参考 https://paddlepaddle.github.io/PaddleOCR/ppocr/blog/ocr_book.html#_2

**Comment by FlyBeeNie** [comment]
下载不了吗

**Reply by GreatV** [reply]
网盘里

**Comment by FlyBeeNie** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
